modules/acquia_cms_common/src/Commands/AcmsConfigImportCommands.php

```diff
@@ -544,4 +618,31 @@ private function hasValidPackage(array $packages): bool {
     return TRUE;
   }
 
+  /**
+   * Execute site studio rebuild after Acquia CMS config reset.
+   *
+   * @hook post-command acms:config-reset
+   */
+  public function postCommand($result, CommandData $commandData) {
```

Comment:
@chandan-singh7929 We are doing the similar thing in _drush/Commands/SiteInstallCommands.php_ i.e. Cohesion rebuild. Please see, if we can reuse code and avoid code duplication. #750

Reply:
Done, as we have refactor the code this increase the test scope of the ticket. Now we have to check site install and see that site studio build is happening after site-install.